webui/react/src/services/decoder.ts

```diff
@@ -147,6 +147,10 @@ export const mapV1Tensorboard =
     };
   };
 
+export const mapV1Template = (template: Sdk.V1Template): types.Template => {
+  return { config: template.config || {}, name: template.name || '' };
```

Comment:
@shiyuann just want to confirm that changing `proto/src/determined/template/v1/template.proto`to the following is ok (making the template `name` and `config` required in the JSON schema):
```
syntax = "proto3";

package determined.template.v1;
option go_package = "github.com/determined-ai/determined/proto/pkg/templatev1";

import "google/protobuf/struct.proto";
import "protoc-gen-swagger/options/annotations.proto";

// Templates move settings that are shared by many experiments into a single
// YAML file.
message Template {
  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
    json_schema: { required: [ "name", "config" ] }
  };
  // The name of the template.
  string name = 1;
  // The template value.
  google.protobuf.Struct config = 4;
}
```

Reply:
@hkang1 @EmilyBonar I think it's ok.